vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -171,26 +171,26 @@ func (calc *UpdatePriorityCalculator) GetSortedPods(admission PodEvictionAdmissi
 func (calc *UpdatePriorityCalculator) GetProcessedRecommendationTargets(r *vpa_types.RecommendedPodResources) string {
 	sb := &strings.Builder{}
 	for _, cr := range r.ContainerRecommendations {
-		sb.WriteString(fmt.Sprintf("%s: ", cr.ContainerName))
+		fmt.Fprintf(sb, "%s: ", cr.ContainerName)
 		if cr.Target != nil {
-			sb.WriteString("target: ")
+			fmt.Fprint(sb, "target: ")
 			if !cr.Target.Memory().IsZero() {
-				sb.WriteString(fmt.Sprintf("%dk ", cr.Target.Memory().ScaledValue(resource.Kilo)))
+				fmt.Fprintf(sb, "%dk ", cr.Target.Memory().ScaledValue(resource.Kilo))
```

Comment:
I think what I meant with my original command was that directly calling strconv.Format functions might be more efficient. Maybe that's over optimizing.
Up to you.